Installation issue: py-pyqt4

<!-- Thanks for taking the time to report this build failure. To proceed with the report please:

1. Title the issue "Installation issue: <name-of-the-package>".
2. Provide the information required below.

We encourage you to try, as much as possible, to reduce your problem to the minimal example that still reproduces the issue. That would help us a lot in fixing it quickly and effectively! -->

### Steps to reproduce the issue

<!-- Fill in the exact spec you are trying to build and the relevant part of the error message -->
```console
$ spack install -v py-pyqt4
...
7 errors found in build log:
  >> 3      Error: Unable to create the C++ code.
     4      Querying qmake about your Qt installation...
     5      Determining the details of your Qt installation...
     6      /home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o qtdetail.mk qtdetail.pro
     7      make -f qtdetail.mk
     8      c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3
            .1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfu
            czxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I.
            -o qtdetail.o qtdetail.cpp
     9      c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o qtdetail qtdetail.o
                -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -L/home/spack/spack/opt/spack/
            linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.
            8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lpthread

     ...

     67     c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_TESTLIB_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack
            /linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gc
            c-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfucz
            xehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtTest
            -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtTest.o cfgtest_QtTest.
            cpp
     68     c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtTest cfgt
            est_QtTest.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtTest -L/home/spack/spac
            k/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc
            -8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/usr/X11R6/lib -lQtCore -lpthread
     69     Checking to see if the QtWebKit module should be built...
     70     /home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtWebKit.mk cfgtest_QtWebKit
            .pro
     71     make -f cfgtest_QtWebKit.mk
     72     c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack
            /spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-ce
            ntos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7
            -pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553
            em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtWebKit -I/home/spa
            ck/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtWebKit.o cfgtest_QtWebKit.cpp
  >> 73     cfgtest_QtWebKit.cpp:1:10: fatal error: qwebpage.h: No such file or directory
     74      #include <qwebpage.h>
     75               ^~~~~~~~~~~~
     76     compilation terminated.
  >> 77     make: *** [cfgtest_QtWebKit.mk:207: cfgtest_QtWebKit.o] Error 1
     78     Checking to see if the QtXml module should be built...
     79     /home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtXml.mk cfgtest_QtXml.pro
     80     make -f cfgtest_QtXml.mk
     81     c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_XML_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aa
            rch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-
            pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o
            /include/QtXml -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtXml.o c
            fgtest_QtXml.cpp
     82     c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtXml cfgte
            st_QtXml.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -L/home/spack/spack/o
            pt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.
            3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread
     83     Checking to see if the QtXmlPatterns module should be built...

     ...

     96     c++ -c -pipe -std=gnu++98 -O2 -D_REENTRANT -Wall -W -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spa
            ck/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos
            8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDesigner -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7
            -pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2
            o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXml -I/home/spack/spa
            ck/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aar
            ch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDesigner.o cfgtest_QtDesigner.cpp
     97     c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtDesigner
            cfgtest_QtDesigner.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -L/home/
            spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-a
            arch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -lQtGui -L/usr/X11R6/lib -lQtCore -lQtDesigner -lpthread
     98     Checking to see if the QAxContainer module should be built...
     99     /home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QAxContainer.mk cfgtest_QAxC
            ontainer.pro
     100    make -f cfgtest_QAxContainer.mk
     101    c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aa
            rch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-
            pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o
            /include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QAxContai
            ner.o cfgtest_QAxContainer.cpp
  >> 102    cfgtest_QAxContainer.cpp:1:10: fatal error: qaxobject.h: No such file or directory
     103     #include <qaxobject.h>
     104              ^~~~~~~~~~~~~
     105    compilation terminated.
  >> 106    make: *** [cfgtest_QAxContainer.mk:205: cfgtest_QAxContainer.o] Error 1
     107    Checking to see if the QtDBus module should be built...
     108    /home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDBus.mk cfgtest_QtDBus.pro
     109    make -f cfgtest_QtDBus.mk
     110    c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_DBUS_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-a
            arch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7
            -pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2
            o/include/QtDBus -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDBus.
            o cfgtest_QtDBus.cpp
  >> 111    cfgtest_QtDBus.cpp:1:10: fatal error: qdbusconnection.h: No such file or directory
     112     #include <qdbusconnection.h>
     113              ^~~~~~~~~~~~~~~~~~~
     114    compilation terminated.
  >> 115    make: *** [cfgtest_QtDBus.mk:221: cfgtest_QtDBus.o] Error 1
     116    Checking to see if the dbus support module should be built...
     117    pkg-config --cflags-only-I --libs dbus-1
     118    DBus v1 does not seem to be installed.
     119    Qt v4.8.7 (Open Source) is being used.
     120    Qt is built as a shared library.
     121    SIP 4.19.21 is being used.

See build log for details:
  /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-build-out.txt

```

### Information on your system

<!-- Please include the output of `spack debug report` -->

<!-- If you have any relevant configuration detail (custom `packages.yaml` or `modules.yaml`, etc.) you can add that here as well. -->

### Additional information
CPU_ARCH_SUFFIX = aarch64
CentOS Linux release 8.2.2004 (Core)
<!-- Please upload the following files. They should be present in the stage directory of the failing build. Also upload any config.log or similar file if one exists. -->
* [spack-build-out.txt]()
* [spack-build-env.txt]()

<!-- Some packages have maintainers who have volunteered to debug build failures. Run `spack maintainers <name-of-the-package>` and @mention them here if they exist. -->

### General information

<!-- These boxes can be checked by replacing [ ] with [x] or by clicking them after submitting the issue. -->
- [ ] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [ ] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers
- [ ] I have uploaded the build log and environment files
- [ ] I have searched the issues of this repo and believe this is not a duplicate

spack-build-out.txt

==> py-pyqt4: Executing phase: 'configure'
==> [2020-10-15-19:13:08.967942] '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8' 'configure-ng.py' '--pyuic4-interpreter' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8' '--sipdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/share/sip/PyQt4' '--stubsdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages/PyQt4' '--verbose' '--confirm-license' '--qmake' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake' '--sip' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip' '--sip-incdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/include/python3.8' '--bindir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/bin' '--destdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages'
Error: Unable to create the C++ code.
Querying qmake about your Qt installation...
Determining the details of your Qt installation...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o qtdetail.mk qtdetail.pro
make -f qtdetail.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o qtdetail.o qtdetail.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o qtdetail qtdetail.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lpthread 
./qtdetail
This is the GPL version of PyQt 4.12.3 (licensed under the GNU General Public
License) for Python 3.8.6 on linux.
Found the license file pyqt-gpl.sip.
Checking to see if the QtGui module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtGui.mk cfgtest_QtGui.pro
make -f cfgtest_QtGui.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtGui.o cfgtest_QtGui.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtGui cfgtest_QtGui.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtHelp module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtHelp.mk cfgtest_QtHelp.pro
make -f cfgtest_QtHelp.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SQL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSql -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtHelp -I. -o cfgtest_QtHelp.o cfgtest_QtHelp.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtHelp cfgtest_QtHelp.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtHelp -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtNetwork -lQtSql -lQtGui -lQtCore -lpthread 
Checking to see if the QtMultimedia module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtMultimedia.mk cfgtest_QtMultimedia.pro
make -f cfgtest_QtMultimedia.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_MULTIMEDIA_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtMultimedia -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtMultimedia.o cfgtest_QtMultimedia.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtMultimedia cfgtest_QtMultimedia.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtMultimedia -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtNetwork module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtNetwork.mk cfgtest_QtNetwork.pro
make -f cfgtest_QtNetwork.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtNetwork.o cfgtest_QtNetwork.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtNetwork cfgtest_QtNetwork.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtNetwork -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtDeclarative module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDeclarative.mk cfgtest_QtDeclarative.pro
make -f cfgtest_QtDeclarative.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_DECLARATIVE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDeclarative -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDeclarative.o cfgtest_QtDeclarative.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtDeclarative cfgtest_QtDeclarative.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtDeclarative -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -lQtSvg -L/usr/X11R6/lib -lQtSql -lQtXmlPatterns -lQtGui -lQtNetwork -lQtCore -lpthread 
Checking to see if the QtScript module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtScript.mk cfgtest_QtScript.pro
make -f cfgtest_QtScript.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtScript.o cfgtest_QtScript.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtScript cfgtest_QtScript.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtScriptTools module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtScriptTools.mk cfgtest_QtScriptTools.pro
make -f cfgtest_QtScriptTools.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_SCRIPTTOOLS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScriptTools -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtScriptTools.o cfgtest_QtScriptTools.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtScriptTools cfgtest_QtScriptTools.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScriptTools -lQtScript -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtOpenGL module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtOpenGL.mk cfgtest_QtOpenGL.pro
make -f cfgtest_QtOpenGL.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtOpenGL -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/usr/X11R6/include -I. -o cfgtest_QtOpenGL.o cfgtest_QtOpenGL.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtOpenGL cfgtest_QtOpenGL.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtOpenGL -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lGL -lpthread 
Checking to see if the QtSql module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtSql.mk cfgtest_QtSql.pro
make -f cfgtest_QtSql.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SQL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSql -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtSql.o cfgtest_QtSql.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtSql cfgtest_QtSql.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtSql -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtSvg module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtSvg.mk cfgtest_QtSvg.pro
make -f cfgtest_QtSvg.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SVG_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSvg -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtSvg.o cfgtest_QtSvg.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtSvg cfgtest_QtSvg.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtSvg -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtTest module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtTest.mk cfgtest_QtTest.pro
make -f cfgtest_QtTest.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_TESTLIB_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtTest -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtTest.o cfgtest_QtTest.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtTest cfgtest_QtTest.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtTest -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtWebKit module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtWebKit.mk cfgtest_QtWebKit.pro
make -f cfgtest_QtWebKit.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtWebKit -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtWebKit.o cfgtest_QtWebKit.cpp
cfgtest_QtWebKit.cpp:1:10: fatal error: qwebpage.h: No such file or directory
 #include <qwebpage.h>
          ^~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QtWebKit.mk:207: cfgtest_QtWebKit.o] Error 1
Checking to see if the QtXml module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtXml.mk cfgtest_QtXml.pro
make -f cfgtest_QtXml.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_XML_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXml -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtXml.o cfgtest_QtXml.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtXml cfgtest_QtXml.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtXmlPatterns module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtXmlPatterns.mk cfgtest_QtXmlPatterns.pro
make -f cfgtest_QtXmlPatterns.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_XMLPATTERNS_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXmlPatterns -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtXmlPatterns.o cfgtest_QtXmlPatterns.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtXmlPatterns cfgtest_QtXmlPatterns.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXmlPatterns -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtNetwork -lQtCore -lpthread 
Checking to see if the phonon module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_phonon.mk cfgtest_phonon.pro
make -f cfgtest_phonon.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_PHONON_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/phonon -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/phonon_compat -I. -o cfgtest_phonon.o cfgtest_phonon.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_phonon cfgtest_phonon.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lphonon -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtDesigner module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDesigner.mk cfgtest_QtDesigner.pro
make -f cfgtest_QtDesigner.mk
c++ -c -pipe -std=gnu++98 -O2 -D_REENTRANT -Wall -W -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDesigner -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXml -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDesigner.o cfgtest_QtDesigner.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtDesigner cfgtest_QtDesigner.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -lQtGui -L/usr/X11R6/lib -lQtCore -lQtDesigner -lpthread 
Checking to see if the QAxContainer module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QAxContainer.mk cfgtest_QAxContainer.pro
make -f cfgtest_QAxContainer.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QAxContainer.o cfgtest_QAxContainer.cpp
cfgtest_QAxContainer.cpp:1:10: fatal error: qaxobject.h: No such file or directory
 #include <qaxobject.h>
          ^~~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QAxContainer.mk:205: cfgtest_QAxContainer.o] Error 1
Checking to see if the QtDBus module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDBus.mk cfgtest_QtDBus.pro
make -f cfgtest_QtDBus.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_DBUS_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDBus -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDBus.o cfgtest_QtDBus.cpp
cfgtest_QtDBus.cpp:1:10: fatal error: qdbusconnection.h: No such file or directory
 #include <qdbusconnection.h>
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QtDBus.mk:221: cfgtest_QtDBus.o] Error 1
Checking to see if the dbus support module should be built...
pkg-config --cflags-only-I --libs dbus-1
DBus v1 does not seem to be installed.
Qt v4.8.7 (Open Source) is being used.
Qt is built as a shared library.
SIP 4.19.21 is being used.
The sip executable is
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip.
These PyQt4 modules will be built: QtCore, QtGui, QtHelp, QtMultimedia,
QtNetwork, QtDeclarative, QtScript, QtScriptTools, QtOpenGL, QtSql, QtSvg,
QtTest, QtXml, QtXmlPatterns, phonon, QtDesigner.
The PyQt4 Python package will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages.
PyQt4 is being built with generated docstrings.
PyQt4 is being built with 'protected' redefined as 'public'.
The Designer plugin will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/plugins/designer.
The PyQt4 PEP 484 stub files will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages/PyQt4.
The PyQt4 .sip files will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/share/sip/PyQt4.
pyuic4, pyrcc4 and pylupdate4 will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/bin.
The interpreter used by pyuic4 is
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8.
Generating the C++ source for the QtCore module...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip -w -n PyQt4.sip -f -n PyQt4.sip -t WS_X11 -t Qt_4_8_7 -B Qt_5_0_0 -o -P -y QtCore.pyi -g -c /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/QtCore -I sip -I /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/sip /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/sip/QtCore/QtCoremod.sip
sip: Deprecation warning: the -B flag is deprecated

> Thanks for the full spec @wwx734925 , I'm really trying to help you out here and I need your help as part of that: can you please post the failing `spack-build-out.txt` log file as I've requested multiple times?
> 
> I can't build it on my mac (the qmake mkspec has the wrong default value).
> 
> I tried building it on a rhel7 system I use but failed. The error messages you posted are just the configure system looking for the different components, and are not relevant. The real error is that a call to a `sip` binary in the `py-sip` package fails without a useful error message, so the installation of py-qt4 fails.
> 
> @adamjstewart @Sinan81 The py-pyqt4 package shows you two as having edited it; have you been able to install it on any contemporary system? I'm not able to help any more on this one. But given that pyqt4's last release was 2 years ago and qt4 was even longer in the past, this might just not be buildable any more.

spack-build-out.txt
<details>
<pre>
==> py-pyqt4: Executing phase: 'configure'
==> [2020-10-15-19:13:08.967942] '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8' 'configure-ng.py' '--pyuic4-interpreter' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8' '--sipdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/share/sip/PyQt4' '--stubsdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages/PyQt4' '--verbose' '--confirm-license' '--qmake' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake' '--sip' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip' '--sip-incdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/include/python3.8' '--bindir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/bin' '--destdir' '/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages'
Error: Unable to create the C++ code.
Querying qmake about your Qt installation...
Determining the details of your Qt installation...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o qtdetail.mk qtdetail.pro
make -f qtdetail.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o qtdetail.o qtdetail.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o qtdetail qtdetail.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lpthread 
./qtdetail
This is the GPL version of PyQt 4.12.3 (licensed under the GNU General Public
License) for Python 3.8.6 on linux.
Found the license file pyqt-gpl.sip.
Checking to see if the QtGui module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtGui.mk cfgtest_QtGui.pro
make -f cfgtest_QtGui.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtGui.o cfgtest_QtGui.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtGui cfgtest_QtGui.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtHelp module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtHelp.mk cfgtest_QtHelp.pro
make -f cfgtest_QtHelp.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SQL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSql -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtHelp -I. -o cfgtest_QtHelp.o cfgtest_QtHelp.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtHelp cfgtest_QtHelp.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtHelp -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtNetwork -lQtSql -lQtGui -lQtCore -lpthread 
Checking to see if the QtMultimedia module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtMultimedia.mk cfgtest_QtMultimedia.pro
make -f cfgtest_QtMultimedia.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_MULTIMEDIA_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtMultimedia -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtMultimedia.o cfgtest_QtMultimedia.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtMultimedia cfgtest_QtMultimedia.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtMultimedia -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtNetwork module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtNetwork.mk cfgtest_QtNetwork.pro
make -f cfgtest_QtNetwork.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtNetwork.o cfgtest_QtNetwork.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtNetwork cfgtest_QtNetwork.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtNetwork -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtDeclarative module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDeclarative.mk cfgtest_QtDeclarative.pro
make -f cfgtest_QtDeclarative.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_DECLARATIVE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDeclarative -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDeclarative.o cfgtest_QtDeclarative.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtDeclarative cfgtest_QtDeclarative.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtDeclarative -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -lQtSvg -L/usr/X11R6/lib -lQtSql -lQtXmlPatterns -lQtGui -lQtNetwork -lQtCore -lpthread 
Checking to see if the QtScript module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtScript.mk cfgtest_QtScript.pro
make -f cfgtest_QtScript.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtScript.o cfgtest_QtScript.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtScript cfgtest_QtScript.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtScriptTools module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtScriptTools.mk cfgtest_QtScriptTools.pro
make -f cfgtest_QtScriptTools.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_SCRIPTTOOLS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScriptTools -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtScriptTools.o cfgtest_QtScriptTools.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtScriptTools cfgtest_QtScriptTools.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScriptTools -lQtScript -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtOpenGL module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtOpenGL.mk cfgtest_QtOpenGL.pro
make -f cfgtest_QtOpenGL.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtOpenGL -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/usr/X11R6/include -I. -o cfgtest_QtOpenGL.o cfgtest_QtOpenGL.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtOpenGL cfgtest_QtOpenGL.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtOpenGL -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lGL -lpthread 
Checking to see if the QtSql module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtSql.mk cfgtest_QtSql.pro
make -f cfgtest_QtSql.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SQL_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSql -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtSql.o cfgtest_QtSql.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtSql cfgtest_QtSql.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtSql -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtSvg module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtSvg.mk cfgtest_QtSvg.pro
make -f cfgtest_QtSvg.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_SVG_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtSvg -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtSvg.o cfgtest_QtSvg.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtSvg cfgtest_QtSvg.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtSvg -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtTest module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtTest.mk cfgtest_QtTest.pro
make -f cfgtest_QtTest.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_TESTLIB_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtTest -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtTest.o cfgtest_QtTest.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtTest cfgtest_QtTest.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtTest -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtGui -L/usr/X11R6/lib -lQtCore -lpthread 
Checking to see if the QtWebKit module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtWebKit.mk cfgtest_QtWebKit.pro
make -f cfgtest_QtWebKit.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtWebKit -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtWebKit.o cfgtest_QtWebKit.cpp
cfgtest_QtWebKit.cpp:1:10: fatal error: qwebpage.h: No such file or directory
 #include <qwebpage.h>
          ^~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QtWebKit.mk:207: cfgtest_QtWebKit.o] Error 1
Checking to see if the QtXml module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtXml.mk cfgtest_QtXml.pro
make -f cfgtest_QtXml.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_XML_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXml -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtXml.o cfgtest_QtXml.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtXml cfgtest_QtXml.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtCore -lpthread 
Checking to see if the QtXmlPatterns module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtXmlPatterns.mk cfgtest_QtXmlPatterns.pro
make -f cfgtest_QtXmlPatterns.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_XMLPATTERNS_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtNetwork -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXmlPatterns -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtXmlPatterns.o cfgtest_QtXmlPatterns.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtXmlPatterns cfgtest_QtXmlPatterns.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXmlPatterns -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtNetwork -lQtCore -lpthread 
Checking to see if the phonon module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_phonon.mk cfgtest_phonon.pro
make -f cfgtest_phonon.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_PHONON_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/phonon -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/phonon_compat -I. -o cfgtest_phonon.o cfgtest_phonon.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_phonon cfgtest_phonon.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lphonon -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -L/usr/X11R6/lib -lQtGui -lQtCore -lpthread 
Checking to see if the QtDesigner module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDesigner.mk cfgtest_QtDesigner.pro
make -f cfgtest_QtDesigner.mk
c++ -c -pipe -std=gnu++98 -O2 -D_REENTRANT -Wall -W -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDesigner -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtXml -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtScript -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDesigner.o cfgtest_QtDesigner.cpp
c++ -Wl,-O1 -Wl,-rpath,/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -o cfgtest_QtDesigner cfgtest_QtDesigner.o    -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtScript -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/openssl-1.0.2u-7ydqegldhr3r2vquvl2gi65s27x3jrvv/lib -L/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/lib -lQtXml -lQtGui -L/usr/X11R6/lib -lQtCore -lQtDesigner -lpthread 
Checking to see if the QAxContainer module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QAxContainer.mk cfgtest_QAxContainer.pro
make -f cfgtest_QAxContainer.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtGui -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QAxContainer.o cfgtest_QAxContainer.cpp
cfgtest_QAxContainer.cpp:1:10: fatal error: qaxobject.h: No such file or directory
 #include <qaxobject.h>
          ^~~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QAxContainer.mk:205: cfgtest_QAxContainer.o] Error 1
Checking to see if the QtDBus module should be built...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/bin/qmake -o cfgtest_QtDBus.mk cfgtest_QtDBus.pro
make -f cfgtest_QtDBus.mk
c++ -c -pipe -std=gnu++98 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_DBUS_LIB -DQT_CORE_LIB -DQT_SHARED -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/mkspecs/linux-g++ -I. -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtCore -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include/QtDBus -I/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/include -I. -o cfgtest_QtDBus.o cfgtest_QtDBus.cpp
cfgtest_QtDBus.cpp:1:10: fatal error: qdbusconnection.h: No such file or directory
 #include <qdbusconnection.h>
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [cfgtest_QtDBus.mk:221: cfgtest_QtDBus.o] Error 1
Checking to see if the dbus support module should be built...
pkg-config --cflags-only-I --libs dbus-1
DBus v1 does not seem to be installed.
Qt v4.8.7 (Open Source) is being used.
Qt is built as a shared library.
SIP 4.19.21 is being used.
The sip executable is
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip.
These PyQt4 modules will be built: QtCore, QtGui, QtHelp, QtMultimedia,
QtNetwork, QtDeclarative, QtScript, QtScriptTools, QtOpenGL, QtSql, QtSvg,
QtTest, QtXml, QtXmlPatterns, phonon, QtDesigner.
The PyQt4 Python package will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages.
PyQt4 is being built with generated docstrings.
PyQt4 is being built with 'protected' redefined as 'public'.
The Designer plugin will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/qt-4.8.7-pgnv6hx2gkpfuczxehpprv4mn553em2o/plugins/designer.
The PyQt4 PEP 484 stub files will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/lib/python3.8/site-packages/PyQt4.
The PyQt4 .sip files will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/share/sip/PyQt4.
pyuic4, pyrcc4 and pylupdate4 will be installed in
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/bin.
The interpreter used by pyuic4 is
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/python-3.8.6-g4ysvcjpqnbw5m3mp3zzt6uqvogpuqel/bin/python3.8.
Generating the C++ source for the QtCore module...
/home/spack/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/py-sip-4.19.21-ibwn5ew34xh2vsvueatwyfse2xw7pukw/bin/sip -w -n PyQt4.sip -f -n PyQt4.sip -t WS_X11 -t Qt_4_8_7 -B Qt_5_0_0 -o -P -y QtCore.pyi -g -c /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/QtCore -I sip -I /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/sip /tmp/root/spack-stage/spack-stage-py-pyqt4-4.12.3-hxagzc23opqpxy3pwp3ffw63ntlibpqs/spack-src/sip/QtCore/QtCoremod.sip
sip: Deprecation warning: the -B flag is deprecated
</pre>
</details>

--- a/var/spack/repos/builtin/packages/py-pyqt4/package.py
+++ b/var/spack/repos/builtin/packages/py-pyqt4/package.py
@@ -33,7 +33,7 @@ class PyPyqt4(SIPPackage):
 
     # Supposedly can also be built with Qt 5 compatibility layer
     depends_on('qt@:4')
-    depends_on('py-sip module=PyQt4.sip')
+    depends_on('py-sip@:4.19.18 module=PyQt4.sip')
 
     # https://www.riverbankcomputing.com/static/Docs/PyQt4/installation.html
     def configure_file(self):

